test/StakeManagement.test.ts

```diff
@@ -280,6 +276,17 @@ contract(
                         from: owner
                     }
                 );
+
+                await relayHub.addRelayWorkers(
+                    ['0x7701f65207DcabA47EbD7826315A47Ec1C6B243d'],
+                    {
+                        from: relayManager
+                    }
+                );
+
+                await relayHub.registerRelayServer('somerelayurl', {
+                    from: relayManager
+                });
```

Comment:
so this causes the following failure:

```
  1) Contract: StakeManagement
       with unlock scheduled
         "before each" hook for "should report relayManager stake as not valid":
     Error: Returned error: VM Exception while processing transaction: revert RelayManager not staked
      at Context.<anonymous> (test/StakeManagement.test.ts:171:28)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:97:5)
``` 

which is even _more_ mystifying to me:
1. why is this a _revert_, instead of just a result of `false` for `relayHub.isRelayManagerStaked(relayManager)`? the method is supposed to be a view
    1. the revert message also appears to be from another function from the relay hub contract
2. why was this error not occurring before?

a more thorough investigation is needed, i'll take another look tomorrow

Reply:
I think these lines can be removed, since that it looks like they have no effect. I tried to remove them and it seems the tests run successfully. Furthermore, we are doing the same operation as what we do at line #340.

Reply:
not sure how i was running this before, but i ran it again and the tests passed. i'll also be on the lookout for the CI results, but for now i've removed these new statements